pcsx2/gui/FrameForGS.cpp

```diff
@@ -438,7 +438,7 @@ GSFrame::~GSFrame() throw()
 void GSFrame::OnCloseWindow(wxCloseEvent& evt)
 {
 	sApp.OnGsFrameClosed( GetId() );
-	Hide();		// and don't close it.
+	CoreThread.Reset(); // reset thread to avoid suspend status on close event.
```

Comment:
@avih 

It seems that even a function to check whether it has been suspended (or) not might not help, since we have already suspended the thread manually before resuming occurs, so I think it's pretty much impossible.